cmd/root/sweet.go

```diff
@@ -495,17 +416,27 @@ func FromArgs(cmd *cobra.Command, args []string) (exercise Exercise, err error)
 			return
 		}
 
+		language, _ := cmd.Flags().GetString("language")
 		var files []os.DirEntry
 		for _, entry := range entries {
 			if entry.IsDir() {
 				continue
 			}
+			// Get the extension of a filename
+			ext := strings.Split(entry.Name(), ".")[1]
```

Comment:
This will probably have to break out into its own function if I want to add a label for ranges of lines to the name of an exercise.